jellyfin_kodi/helper/loghandler.py

```diff
@@ -65,15 +65,15 @@ def emit(self, record):
 
             if self.mask_info:
                 for server in self.sensitive['Server']:
-                    string = string.replace(server.encode('utf-8') or "{server}", "{jellyfin-server}")
+                    string = string.replace(server or "{server}", "{jellyfin-server}")
 
                 for token in self.sensitive['Token']:
-                    string = string.replace(token.encode('utf-8') or "{token}", "{jellyfin-token}")
+                    string = string.replace(token or "{token}", "{jellyfin-token}")
 
             try:
                 xbmc.log(string, level=xbmc.LOGNOTICE)
             except UnicodeEncodeError:
```

Comment:
Can this try except not be removed now?

Reply:
I have absolutely no idea.
I did a search and replace on `\.(en|de)code\(['"]utf-8['"]\)` and killed 99% of them.

Probably removed a few too many.